frontend/src/pages/norms/eli/[jurisdiction]/[agent]/[year]/[naturalIdentifier]/[pointInTime]/[version]/[language]/[subtype]/index.vue

```diff
@@ -139,11 +140,10 @@ const breadcrumbItems: ComputedRef<BreadcrumbItem[]> = computed(() => {
           <FileActionsMenu :xml-url="xmlUrl" />
         </div>
         <NormHeadingGroup :metadata="metadata" :html-parts="htmlParts" />
-
-        <VersionWarningMessage
+        <VersionWarning
           v-if="normVersionsStatus === 'success'"
           :versions="normVersions"
-          :current-expression="metadata.workExample.legislationIdentifier"
+          :current-version="metadata"
```

Comment:
I think we should agree on a common word for expression/version. I used expression over version in my changes. And I noticed its kinda mixed all over the place 😅